app/ui/frontend/src/components/ResumeForm.vue

```diff
@@ -96,20 +123,53 @@ export default {
         document.getElementById("#job_cards").scrollIntoView();
       })
     },
-    job_openings_info: function(job_name) {
-      var base_url = "http://34.235.155.212:5601/app/kibana#/dashboard/1f46d3b0-36eb-11e8-bc51-3d1df9db5706?_g=()&_a=(filters:!(),options:(darkTheme:!f),panels:!((col:7,id:'901a10a0-36e8-11e8-bc51-3d1df9db5706',panelIndex:1,row:1,size_x:6,size_y:3,type:visualization),(col:1,id:f2ba0fc0-36e9-11e8-bc51-3d1df9db5706,panelIndex:2,row:4,size_x:6,size_y:3,type:visualization),(col:7,id:'2133c180-36e9-11e8-bc51-3d1df9db5706',panelIndex:3,row:4,size_x:6,size_y:3,type:visualization),(col:1,id:'51128180-36e8-11e8-bc51-3d1df9db5706',panelIndex:4,row:1,size_x:6,size_y:3,type:visualization),(col:1,id:'9d9c34a0-36e9-11e8-bc51-3d1df9db5706',panelIndex:5,row:7,size_x:6,size_y:3,type:visualization),(col:7,id:'79876e00-3a30-11e8-8de6-81a699545c9e',panelIndex:6,row:7,size_x:6,size_y:3,type:visualization),(col:1,id:ce3585d0-3a7c-11e8-a300-439e4674ea29,panelIndex:7,row:10,size_x:6,size_y:3,type:visualization)),query:(query_string:(analyze_wildcard:!t,REPLACE_QUERY_TERM)),timeRestore:!f,title:'Jobs%20Dashboard',uiState:(),viewMode:view)";
+    show_jobs: function(job_name) {
+      var base_url = "http://34.235.155.212:5601/app/kibana#/dashboard/dd5be340-4078-11e8-878a-e7a4605920c3?embed=true&_g=()&_a=(filters:!(),options:(darkTheme:!f),panels:!((col:1,id:f2ba0fc0-36e9-11e8-bc51-3d1df9db5706,panelIndex:1,row:1,size_x:6,size_y:3,type:visualization),(col:7,id:'9d9c34a0-36e9-11e8-bc51-3d1df9db5706',panelIndex:2,row:4,size_x:6,size_y:3,type:visualization),(col:7,id:'2133c180-36e9-11e8-bc51-3d1df9db5706',panelIndex:3,row:1,size_x:6,size_y:3,type:visualization),(col:1,id:'901a10a0-36e8-11e8-bc51-3d1df9db5706',panelIndex:4,row:4,size_x:6,size_y:3,type:visualization)),query:(query_string:(analyze_wildcard:!t,REPLACE_QUERY_TERM)),timeRestore:!f,title:Jobs,uiState:(),viewMode:view)";
+
+      var url_encoded_job_name = encodeURIComponent(job_name);
+      var query_param = "query:'jobType:%20%22".concat(url_encoded_job_name, "%22%20OR%20searchTerm:%20%22", url_encoded_job_name, "%22'");
+      var url = base_url.replace("REPLACE_QUERY_TERM", query_param);
+
+      var iframe_html = '<iframe src="' + url + '" frameborder="0" style="height:1200px;width:100%;display:block;" height="100%" width="100%"></iframe>';
+
+      this.$refs.jobsModalRef.show();
```

Comment:
there is potential for refactor & cleanup here. But, in the interest of time, I wanted to get in stuff that works.